src/main/java/org/ladocuploader/app/preparers/ExpeditedSnapPreparer.java

```diff
@@ -0,0 +1,83 @@
+package org.ladocuploader.app.preparers;
+
+import static java.util.Collections.emptyList;
+
+import formflow.library.data.Submission;
+import formflow.library.pdf.PdfMap;
+import formflow.library.pdf.SingleField;
+import formflow.library.pdf.SubmissionField;
+import formflow.library.pdf.SubmissionFieldPreparer;
+import java.util.HashMap;
+import java.util.Map;
+import org.springframework.stereotype.Component;
+
+@Component
+public class ExpeditedSnapPreparer implements SubmissionFieldPreparer {
+
+  @Override
+  public Map<String, SubmissionField> prepareSubmissionFields(Submission submission, PdfMap pdfMap) {
+    Map<String, SubmissionField> results = new HashMap<>();
+
+    String isEligibleForExpeditedSnap = submission.getInputData().get("isEligibleForExpeditedSnap").toString();
+    if (isEligibleForExpeditedSnap.equals("true")) {
+      String householdIncomeLast30Days=submission.getInputData().getOrDefault("householdIncomeLast30Days", "0").toString();
+      results.put("expeditedSnapIncome",  new SingleField("expeditedSnapIncome", householdIncomeLast30Days, null));
+
+      String moneyOnHandString = submission.getInputData().getOrDefault("expeditedMoneyOnHandAmount", "0").toString();
+      results.put("expeditedSnapMoneyOnHand",  new SingleField("expeditedSnapMoneyOnHand", moneyOnHandString, null));
+
+      String householdRentAmount = submission.getInputData().getOrDefault("householdRentAmount", "0").toString();
+      results.put("expeditedSnapHousingCost",  new SingleField("expeditedSnapHousingCost", householdRentAmount, null));
+
+      var utilities = processUtilitiesExpenses(submission);
+      boolean hasHouseholdExpenses = utilities.get("expenses");
+      boolean hasHeatingExpenses = utilities.get("acOrCooling");
```

Comment:
Could be slightly more readable to separate these into their own methods that return just the boolean

``` java
boolean hasHouseholdExpenses = hasHouseholdExpenses(submission);
boolean hasHeatingExpenses = hasHeatingExpenses(submission);
```